src/react-components/src/__tests__/components/FormDropdown-test.tsx

```diff
@@ -0,0 +1,54 @@
+import { expect } from "chai";
+import { stub } from "sinon";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+
+import * as Mocha from "mocha";
+
+import FormDropdown from "../../components/01-atoms/Forms/FormDropdown";
+
+describe("FormDropdown", () => {
+  let wrapper: Enzyme.ShallowWrapper<{}, {}>;
+  let blurCallback;
+  let changeCallback;
+
+  beforeEach(() => {
+    changeCallback = stub();
+    blurCallback = stub();
+  });
+
+  it("Form Dropdown", () => {
+    wrapper = Enzyme.shallow(<FormDropdown dropdownId="hi" labelText="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />);
+    expect(wrapper.find("select")).to.have.lengthOf(1);
+  });
+
+  it("sends callback with value on change", () => {
+    wrapper = Enzyme.shallow(<FormDropdown
+      dropdownId="hi" labelText="hi"
+      options={["opt1", "opt2", "opt3"]}
+      onSelectChange={changeCallback}
+      onSelectBlur={blurCallback} />);
+
+    wrapper.find("select").simulate("change", "", { value: ["val"] });
+
+    expect(changeCallback.callCount).to.equal(1);
+  });
+
+  it("sends callback with no value on blur", () => {
+    wrapper = Enzyme.shallow(<FormDropdown dropdownId="hi" labelText="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />);
+    wrapper.find("select").simulate("blur");
+
+    expect(blurCallback.callCount).to.equal(1);
+  });
+
+  it("requires either label or aria-label", () => {
+    expect(() => Enzyme.mount(<FormDropdown dropdownId="hi" options={["opt1", "opt2", "opt3"]} onSelectChange={changeCallback} onSelectBlur={blurCallback} />))
+    .to.throw("Either labelText or ariaLabel must be defined");
+  });
+
+  it("renders empty select form when passed no options", () => {
```

Comment:
I think it's more about thinking on how it will be used.  Ideally, I want the person consuming this to have tests on their end making sure that their options lists aren't ever empty.  However, I also think it's overly punishing to throw an error / not render at all if the user runs into a data situation where there's nothing in the list ... thoughts?

Reply:
I'm with @hokei on this one—I don't think we want the components to not appear in the library itself, I think that should be handled by the apps on integration in their business logic.

Reply:
That sounds fine. In general, I find it hard to trust another developer to use a component as it should be use and could be misused without clear examples or documentation.

Reply:
That's also a solid reason. I'm going to vote we move forward with this as it is and I'm going to file this conversation in the backlog to revisit at a later date.

Reply:
+1 to having clear documentation as to how this should be used.  Not sure where something like that would live, yet, but we're gonna start piling up best practices like this, that aren't explicitly checked for in the code.